<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-IA0DLJF" name="TestChrome" tests="4" failures="4" timestamp="7 May 2019 03:37:20 GMT" time="532.841" errors="0">
  <testcase name="TC1" time="60.176" classname="test.java.com.pulse.Test.ValidLogin1">
    <failure type="java.lang.NoSuchMethodError" message="com.aventstack.extentreports.ExtentTest.assignCategory(Ljava/lang/String;)Lcom/aventstack/extentreports/ExtentTest;">
      <![CDATA[java.lang.NoSuchMethodError: com.aventstack.extentreports.ExtentTest.assignCategory(Ljava/lang/String;)Lcom/aventstack/extentreports/ExtentTest;
	at test.java.com.pulse.Test.ValidLogin1.TC1(ValidLogin1.java:83)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- TC1 -->
  <testcase name="TC2" time="83.363" classname="test.java.com.pulse.Test.ValidLogin1">
    <failure type="java.lang.AssertionError" message="expected [Facebook ï¿½ log in or sign up] but found [Facebook – log in or sign up]">
      <![CDATA[java.lang.AssertionError: expected [Facebook ï¿½ log in or sign up] but found [Facebook – log in or sign up]
	at test.java.com.pulse.Test.ValidLogin1.TC2(ValidLogin1.java:132)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC2 -->
  <testcase name="TC3" time="0.003" classname="test.java.com.pulse.Test.ValidLogin2">
    <failure type="java.lang.NoSuchMethodError" message="com.aventstack.extentreports.ExtentTest.assignCategory(Ljava/lang/String;)Lcom/aventstack/extentreports/ExtentTest;">
      <![CDATA[java.lang.NoSuchMethodError: com.aventstack.extentreports.ExtentTest.assignCategory(Ljava/lang/String;)Lcom/aventstack/extentreports/ExtentTest;
	at test.java.com.pulse.Test.ValidLogin2.TC3(ValidLogin2.java:82)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- TC3 -->
  <testcase name="TC4" time="74.854" classname="test.java.com.pulse.Test.ValidLogin2">
    <failure type="java.lang.NoSuchMethodError" message="com.aventstack.extentreports.ExtentTest.assignCategory(Ljava/lang/String;)Lcom/aventstack/extentreports/ExtentTest;">
      <![CDATA[java.lang.NoSuchMethodError: com.aventstack.extentreports.ExtentTest.assignCategory(Ljava/lang/String;)Lcom/aventstack/extentreports/ExtentTest;
	at test.java.com.pulse.Test.ValidLogin2.TC4(ValidLogin2.java:128)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- TC4 -->
</testsuite> <!-- TestChrome -->
